example/binary_request_execution/README.md

```diff
@@ -0,0 +1,74 @@
+# Binary request execution sample
+This sample code is to demonstrate that the node sdk can be used to execute Scalar DL binary request build by another application using any Scalar sdk.
+
+![](img/samplecode.png)
+
+In conclusion, the listContracts request sent by Web SDK can be executed by ScalarDLT invoked from node sdk.
+
+## Project Prerequisite
+Make sure your ScalarDLT server is up and running by referring to [Scalar DL official documentation](https://scalardl.readthedocs.io/en/latest/). 
+
+`ClientProperties` and `WebService` can be initiated as follows:
+```
+// web-client
```

Comment:
After all is done, please update the clientProperties below to reflect the changes on the code